packages/core/src/shared/elementUtils.ts

```diff
@@ -0,0 +1,36 @@
+import type { ComponentPublicInstance } from 'vue'
+
+/**
+ * Check if the target is a HTMLElement
+ * @param target - The target to check
+ * @returns True if the target is a HTMLElement, false otherwise
+ */
+export function isHTMLElement(target: unknown): target is HTMLElement {
+  return !!target && target instanceof HTMLElement
```

Comment:
The `!!target` gate is unnecessary; null and undefined are valid operands of instanceof and will produce false as expected